packages/web/src/ui/flow/design.js

```diff
@@ -478,21 +477,6 @@ const actions = ({ sources }) => {
   const setDesignSolids$ = most.mergeArray([
     sources.solidWorker
       .filter((event) => !('error' in event) && event.data instanceof Object && event.data.type === 'solids')
-      .map((event) => {
-        try {
-          const start = new Date().getTime()
-
-          const solids = deserializeSolids(event.data.solids)
-
-          const end = new Date().getTime()
-          console.warn(`elapse for solid generation: ${end - start}`)
-
-          const { lookupCounts, lookup } = event.data
-          return { solids, lookup, lookupCounts }
```

Comment:
The return from this map((event) is different now. Does this work?

Reply:
you are right, it stopped working when this was completely removed. I have made a fix